src/NzbDrone.Core/MediaFiles/EpisodeImport/ImportDecisionMaker.cs

```diff
@@ -15,7 +17,8 @@ namespace NzbDrone.Core.MediaFiles.EpisodeImport
 {
     public interface IMakeImportDecision
     {
-        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series, bool sceneSource, QualityModel quality = null);
+        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series);
+        List<ImportDecision> GetImportDecisions(List<String> videoFiles, Series series, bool sceneSource, ParsedEpisodeInfo folderInfo);
```

Comment:
also, swap the position of `ParsedEpisodeInfo` and `SceneSource`, flag should always be last. (you can use re-sharper to change the order)